packages/gasket-plugin-nextjs/package.json

```diff
@@ -109,5 +104,8 @@
     "presets": [
       "@babel/preset-react"
     ]
+  },
+  "jest": {
+    "rootDir": "test"
   }
```

Comment:
Why is this needed?

Reply:
Because it's going into the generator directory

Reply:
👍 got it